tests/native/test_armv7cpu.py

```diff
@@ -13,25 +13,266 @@
 from manticore.native.memory import SMemory32
 from manticore.utils.helpers import pickle_dumps
 
-ks = Ks(KS_ARCH_ARM, KS_MODE_ARM)
-ks_thumb = Ks(KS_ARCH_ARM, KS_MODE_THUMB)
+ks = None
+ks_thumb = None
 
 import logging
 
 logger = logging.getLogger("ARM_TESTS")
 solver = Z3Solver.instance()
 
+assembly_cache = {
```

Comment:
```suggestion
# This is a cache of assembled instructions.
# This exists so that Manticore's tests can run without requiring that the
# Keystone dependency be installed.
# If additional test cases are added that require new instructions, this cache
# will need to be updated.
assembly_cache = {
```